ui/src/features/cockpit/hooks/useCockpitState.ts

```
@@ -0,0 +1,65 @@
+
+  // Auto-select first workspace on initial load, or create "default" if none exist
+  useEffect(() => {
+    if (!data) return;
+    if (selectedWorkspace) return;
+    if (data.workspaces?.length) {
+      setSelectedWorkspace(data.workspaces[0]!.name);
+    } else if (!autoCreatedRef.current) {
+      autoCreatedRef.current = true;
+      createWorkspace('default');
+    }
+  }, [data?.workspaces, createWorkspace]);
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)